### ISSUE TITLE
Style not showing up

### ISSUE THREAD

**Issue by ridmeh** [xeodou/react-crouton#9]
While, trying to use it I am unable to get styles shown. All text comes with white background. What could be the reason or any direction where I can look at possible issue?
I have intergrated the provided styles.css using following line in html code.
<link rel="stylesheet" type="text/css" href="./style.css">

**Comment by ridmeh** [comment]
`<link rel="stylesheet" type="text/css" href="./style.css">`

Great work.

**Comment by ridmeh** [comment]
i meant to write, text is shown in white. Changed the following (in styles file) for testing purpose and it shows in black. However, no border and other things are shown. Am I using right css file?
Changed from:
.crouton {
  color: #FFF;
to:
.crouton {
  color: $black;

**Comment by xeodou** [comment]
Hi @ridmeh , are you using the `style.css` file contains inside of example ? The component doesn't provide the styles itself.

**Comment by ridmeh** [comment]
yes. I was following the e.g. Do I not need to provide styles?

**Comment by ridmeh** [comment]
I have also tried without providing styles, It does not draw the box and puts respective colors.

**Comment by xeodou** [comment]
yes, I wrote the css for the example and it doesn't provide in the component, but you can still take look build your own styles.
